### ISSUE TITLE
onfigure virtual line sensor in lidar loc software

### ISSUE THREAD

**Issue by JonasPetersenFAU** [amosproj/amos2022ss03-turtlebot-fleet-management#22]
## User story
1. As a SD
2. I want / need to configure the virtual line sensor in Lidar Loc
3. So that we can recieve a virtual line measurement 

Setup
Sensors
Virtual line sensors   Communication tab for ip and port
default values

## Acceptance criteria
* Criterion 1
* Criterion 2
* ...

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
